README.md

```diff
@@ -12,43 +12,72 @@ version,even the default kubernetes cluster configuration and so on.
 auto-build only accept one arg that is image version,and will use this version to pull related kubernetes container
 images, so make sure it is a valid value.
 
+```shell
+## auto-build options:
+  -v, --version         set the kubernetes version of the cluster image, version must be greater than 1.13
```

Comment:
ClusterImage is a reserved word in sealer's concept. I wish we could unified to use `ClusterImage`.

Reply:
In addition, `-v` flag is always used to the binary itself, which means the auto-build tool version. maybe you can use k8s-version to make it more explicit.